dev/kyuubi-extension-spark-3-1/src/test/scala/org/apache/spark/sql/KyuubiExtensionSuite.scala

```diff
@@ -1026,4 +1046,402 @@ class KyuubiExtensionSuite extends QueryTest with SQLTestUtils with AdaptiveSpar
+    auxiStatementSimpleName.add(
+      spark.sessionState.analyzer.execute(
+        spark.sessionState.sqlParser.parsePlan(sql34)
+      ).getClass.getSimpleName
+    )
+
+    val sql36 = "SET spark.sql.variable.substitute=false"
+    auxiStatementSimpleName.add(
+      spark.sessionState.analyzer.execute(
+        spark.sessionState.sqlParser.parsePlan(sql36)
+      ).getClass.getSimpleName
+    )
+
+    val sql37 = "SET"
+    auxiStatementSimpleName.add(
+      spark.sessionState.analyzer.execute(
+        spark.sessionState.sqlParser.parsePlan(sql37)
+      ).getClass.getSimpleName
+    )
+
+    val sql38 = "SET spark.sql.variable.substitute"
+    auxiStatementSimpleName.add(
+      spark.sessionState.analyzer.execute(
+        spark.sessionState.sqlParser.parsePlan(sql38)
+      ).getClass.getSimpleName
+    )
+
+    val sql39 = "RESET"
+    auxiStatementSimpleName.add(
+      spark.sessionState.analyzer.execute(
+        spark.sessionState.sqlParser.parsePlan(sql39)
+      ).getClass.getSimpleName
+    )
+
+    val sql40 = "RESET spark.sql.variable.substitute"
+    auxiStatementSimpleName.add(
+      spark.sessionState.analyzer.execute(
+        spark.sessionState.sqlParser.parsePlan(sql40)
+      ).getClass.getSimpleName
+    )
+
+    val sql41 = "SET TIME ZONE LOCAL;"
+    auxiStatementSimpleName.add(
+      spark.sessionState.analyzer.execute(
+        spark.sessionState.sqlParser.parsePlan(sql41)
+      ).getClass.getSimpleName
+    )
+
+    val sql42 = "ADD FILE /tmp/test;"
+    auxiStatementSimpleName.add(
+      spark.sessionState.analyzer.execute(
+        spark.sessionState.sqlParser.parsePlan(sql42)
+      ).getClass.getSimpleName
+    )
+
+    val sql43 = "ADD JAR /tmp/test.jar;"
+    auxiStatementSimpleName.add(
+      spark.sessionState.analyzer.execute(
+        spark.sessionState.sqlParser.parsePlan(sql43)
+      ).getClass.getSimpleName
+    )
+
+    val sql44 = "LIST FILE;"
+    auxiStatementSimpleName.add(
+      spark.sessionState.analyzer.execute(
+        spark.sessionState.sqlParser.parsePlan(sql44)
+      ).getClass.getSimpleName
+    )
+
+    val sql45 = "LIST JAR;"
+    auxiStatementSimpleName.add(
+      spark.sessionState.analyzer.execute(
+        spark.sessionState.sqlParser.parsePlan(sql45)
+      ).getClass.getSimpleName
+    )
+
+    val sql46 = "EXPLAIN select k, sum(v) from values (1, 2), (1, 3) t(k, v) group by k;"
+    auxiStatementSimpleName.add(
+      spark.sessionState.analyzer.execute(
+        spark.sessionState.sqlParser.parsePlan(sql46)
+      ).getClass.getSimpleName
+    )
+
+    val sql47 = "EXPLAIN EXTENDED select k, sum(v) from values (1, 2), (1, 3) t(k, v) group by k;"
+    auxiStatementSimpleName.add(
+      spark.sessionState.analyzer.execute(
+        spark.sessionState.sqlParser.parsePlan(sql47)
+      ).getClass.getSimpleName
+    )
+
+    // scalastyle:off println
+    println("auxiliary statement simple name is :" + auxiStatementSimpleName.toSeq.sorted)
```

Comment:
what does this test want to do ? not see any assert

Reply:
this ut is just for me to check the plan and get their simpleName.
Then I use those result set to generat the jsonFile.